client/src/components/Navbar/index.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-unused-vars */
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import {
+  ListItem, ListItemText, Button, Avatar, Container, List,
+} from '@mui/material';
+import './style.css';
+
+import Logo from '../../asstes/logo.png';
+import PresonImg from '../../asstes/avatar.png';
+
+function NavBar() {
+  const [logged, setLogged] = useState(true);
+  return (
+    <Container maxWidth="xl">
+      <nav className="container">
+        <Link to="/">
+          <img src={Logo} alt="logo-housy" className="img-logo" />
+        </Link>
+        {' '}
+        <List
+          className="list"
+          sx={{ justifyContent: 'space-between' }}
+          style={{
+            marginRight: '150px', alignSelf: 'center', color: '#797B7D',
+          }}
+        >
+          <Link to="/" style={{ color: '#797B7D', textDecoration: 'none' }}>
```

Comment:
Try and make all of this
```jsx
<Link to="/signup" style={{ color: '#797B7D', textDecoration: 'none' }}>
  <ListItem button>
    <ListItemText primary="About" />
  </ListItem>
</Link>
```
Into one component, and reuse it here as much as you'd like.